src/content/docs/de/contribute.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Zu Astro beitragen
+description: Wie du dich engagieren und zu Astro beitragen kannst. 
+i18nReady: true
```

Comment:
Not sure if this belongs in the translate files. Feedback appreciated.

Reply:
There's no problem with it being or not present, we only check it for the source (English) pages.

Reply:
Makes sense. Thank you for clarification.